posthog/temporal/batch_exports/spmc.py

```diff
@@ -580,6 +596,9 @@ async def start(
         else:
             filters_str, extra_query_parameters = "", extra_query_parameters
 
+        # TODO: this can be simplified once all backfill inputs are migrated
+        is_backfill = (backfill_details is not None) or is_backfill
+
```

Comment:
```suggestion
        # TODO: this can be simplified once all backfill inputs are migrated
        is_backfill = backfill_details.get_is_backfill()

```